index.html

```diff
@@ -12,5 +13,78 @@
 
 <body>
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+<div class="container">
+  <header>
+    <nav>
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </nav>
+    <img src="img/logo.png" alt="Great Idea! Company logo." id="logo">
+</header>
+
+  
+ 
+  
+<div class="sub-head">
+  <div class="sign">Innovation<br>
+    On<br>
+    <p>Demand<br></p>
+    <button class="button">Get Started</button>
+  </div>
+  <img class="circle" src="img/header-img.png" alt="Image of a code snippet.">
+</div>
+  
+
+
+  <hr class="dash">
```

Comment:
this is not needed you can use a border-top on the section or a border-bottom on the div above